apps/bfd-model/bfd-model-rif/pom.xml

```diff
@@ -72,7 +72,7 @@
 		<dependency>
 			<groupId>org.apache.poi</groupId>
 			<artifactId>poi</artifactId>
-			<version>4.1.1</version>
+			<version>5.2.2</version>
```

Comment:
just curious when I see the poi jar file being part of the deployed SNAPSHOT; why do we package up .jar dependencies that are only needed for the code generation within maven?

Reply:
@christrotteradhoc @lsmitchell any ideas?